<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myFinanceExchangeWindow</class>
 <widget class="QDialog" name="myFinanceExchangeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>资产变化</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>300</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>9</y>
     <width>287</width>
     <height>95</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="horizontalSpacing">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="timeLabel">
      <property name="text">
       <string>交易时间</string>
      </property>
      <property name="buddy">
       <cstring>timeDateTimeEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDateTimeEdit" name="timeDateTimeEdit">
      <property name="displayFormat">
       <string>yyyy-M-d HH:mm:ss</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelType">
      <property name="text">
       <string>交易类别</string>
      </property>
      <property name="buddy">
       <cstring>typeLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="typeLineEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelExchangeFee">
      <property name="text">
       <string>交易费（元）：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>exchangeFeeSpinBox</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="exchangeFeeSpinBox">
      <property name="minimum">
       <double>-100000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100000.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>691</width>
     <height>211</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxRollback">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>是否撤回</string>
   </property>
  </widget>
  <zorder>formLayoutWidget</zorder>
  <zorder>buttonBox</zorder>
  <zorder>tabWidget</zorder>
  <zorder>checkBoxRollback</zorder>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>timeDateTimeEdit</tabstop>
  <tabstop>typeLineEdit</tabstop>
  <tabstop>exchangeFeeSpinBox</tabstop>
  <tabstop>checkBoxRollback</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>myFinanceExchangeWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>408</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>myFinanceExchangeWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
